examples/select-features.js

```diff
@@ -43,6 +43,16 @@ var selectPointerMove = new ol.interaction.Select({
   condition: ol.events.condition.pointerMove
 });
 
+var selectAltClick = new ol.interaction.Select({
+  condition: olConditionClubber(ol.events.condition.altKeyOnly,
+      ol.events.condition.click)
```

Comment:
I had kept the function returning a function to show the possibility of adding more conditions. But yes having it as you've shown would work fine too. I'll incorporate and update the pull request.